src/open_inwoner/openklant/tests/helpers.py

```diff
@@ -0,0 +1,49 @@
+import os
+
+from django.test import TestCase as DjangoTestCase
+
+from typing_extensions import TypedDict
+from vcr.record_mode import RecordMode
+from vcr.unittest import VCRMixin
+
+from openklant2.tests.helpers import LiveOpenKlantTestMixin
+
+
+class OpenKlant2ServiceTestConfig(TypedDict):
+    use_live_service: bool
+    vcr_record_mode: RecordMode
+
+
+class Openklant2ServiceTestCase(VCRMixin, LiveOpenKlantTestMixin, DjangoTestCase):
+
+    vcr_record_mode: RecordMode = RecordMode.NONE
+
+    @classmethod
+    def get_config(cls) -> OpenKlant2ServiceTestConfig:
+        return {
+            "use_live_service": bool(os.getenv("RECORD_OPENKLANT_CASSETTES"))
```

Comment:
For `RECORD_OPENKLANT_CASSETTES=False`, `bool(os.getenv("RECORD_OPENKLANT_CASSETTES"))` will evaluate to `True`. I think that's confusing.